new_arrivals_chi/app/main.py

```diff
@@ -48,7 +49,12 @@ def home():
     """
 
     language = request.args.get("lang", "en")
-    return render_template("home.html", language=language)
+    translations = load_translations()
+    translations = translations[language]
+
+    return render_template(
+        "home.html", language=language, translations=translations
```

Comment:
To save on HTTP response time, I'd only send the translations that apply. For this page it'd be `translations["home"]`.